### PR TITLE
Add the Vert.x Cache booster

### PR DESCRIPTION
**Pull request by cescoffier**

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 missions:
+- id: cache
+  name: Cache
+  description: The Caching mission demonstrates how to use a cache to improve the response time of applications.
```
Comment (Ladicek) [inline metadata.yaml]: So is it "Cache" or "Caching"? :-)

File: metadata.yaml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 missions:
+- id: cache
+  name: Cache
+  description: The Caching mission demonstrates how to use a cache to improve the response time of applications. 
+  metadata:
+    level: foundational
```
Comment (Ladicek) [inline metadata.yaml]: advanced

File: metadata.yaml [hunk 3]
```diff
@@ -18,7 +23,7 @@ missions:
   name: Health Check
   description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
   metadata:
-    level: foundational
+    level: foundational
```
Comment (Ladicek) [inline metadata.yaml]: Ugh...

File: vert.x/community/cache/booster.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+metadata:
+  runsOn: 
+    - '!starter'
+    - '!osio'
+name: Vert.x Cache Booster
+description: Demonstrate how to use a cache server from an Eclipse Vert.x application
```
Comment (Ladicek) [inline vert.x/community/cache/booster.yaml]: Should mention that it's a "JDG cache server"?

File: metadata.yaml (hunk 1 above)
Comment (gastaldi) [reply metadata.yaml]: Agree, I think it should be `The Cache mission ...`

File: vert.x/community/cache/booster.yaml (hunk 4 above)
Comment (gastaldi) [reply vert.x/community/cache/booster.yaml]: The name and description are not used anywhere, they can be safely removed

File: vert.x/community/cache/booster.yaml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    source:
+      git:
+        ref: v1
+  production:
+    source:
+      git:
+        ref: v1
+    metadata:
+      version:
+        name: 3.5.0.Final (Community)
```
Comment (gastaldi) [inline vert.x/community/cache/booster.yaml]: This is already in `metadata.yaml`, can be removed

File: metadata.yaml (hunk 1 above)
Comment (Ladicek) [reply metadata.yaml]: And here https://github.com/fabric8-launcher/launcher-documentation/wiki, we call it _Cache Service_. We're the kings of consistency.

File: vert.x/community/cache/booster.yaml (hunk 4 above)
Comment (cescoffier) [reply vert.x/community/cache/booster.yaml]: no, JDG should not be mentioned.

File: metadata.yaml (hunk 1 above)
Comment (cescoffier) [reply metadata.yaml]: :-)

I prefer Cache.

**Comment by cescoffier** [comment]
I've updated the PR.

File: metadata.yaml (hunk 3 above)
Comment (gastaldi) [reply metadata.yaml]: @cescoffier mind not changing this line? ;)

**Comment by Ladicek** [comment]
LGTM barring the extraneous `metadata.level` change for the Health Check mission.

**Pull request merged**